Core/include/Acts/Geometry/MaterialDesignatorBlueprintNode.hpp

```diff
@@ -78,48 +83,50 @@ class MaterialDesignatorBlueprintNode final : public BlueprintNode {
   /// @note This method can be called multiple times to configure different faces.
   /// @param face The face of the cylinder to configure
   /// @param loc0 The first binning configuration along local axis 0
+  /// @param aDir0 The direction of the first binning configuration
   /// @param loc1 The second binning configuration along local axis 1
+  /// @param aDir1 The direction of the second binning configuration
   /// @return The material designator node
   /// @note If this node has previously been configured with a different volume
   ///       shape, this will throw an exception.
   MaterialDesignatorBlueprintNode& configureFace(
       CylinderVolumeBounds::Face face, const ProtoAxis& loc0,
-      const ProtoAxis& loc1);
+      AxisDirection aDir0, const ProtoAxis& loc1, AxisDirection aDir1);
```

Comment:
Maybe these should still accept `DirectedProtoAxis` (in case you change that to a proper type)?